src/components/molecules/card/card.scss

```
@@ -5,48 +5,42 @@
   height: auto;
   border-radius: 0%;
   border: 0px;
-}
-
-.triangle {
-  position: absolute;
-  z-index: 2;
-  border-bottom: 150px solid transparent;
-  border-left: 150px solid white;
-}
-
-.triangle-text {
-  position: absolute;
-  z-index: 2;
-  font-weight: 700;
-  font-size: 20px;
-  display: flex;
-  align-items: center;
-  color: $orange_bistre;
-  margin-left: 6px;
-}
-
-.card-img-change {
-  transition: all 0.5s ease;
-  cursor: pointer;
-  &:hover {
-    @media screen and(min-width:$breakpoint-md) {
-      transform: scale(1.2, 1.2);
+  .triangle {
+    position: absolute;
+    z-index: 2;
+    border-bottom: 150px solid transparent;
+    border-left: 150px solid white;
+  }
+  .triangle-text {
+    position: absolute;
+    z-index: 2;
+    font-weight: 700;
+    font-size: 20px;
+    display: flex;
+    align-items: center;
+    color: $orange_bistre;
+    margin-left: 6px;
+  }
+  .card-img-change {
+    transition: all 0.5s ease;
+    cursor: pointer;
+    &:hover {
+      @media screen and(min-width:$breakpoint-md) {
+        transform: scale(1.2, 1.2);
```

```suggestion
        transform: scale(1.2);
```
縦横の拡大比率が同じな時はこの書き方で問題ないです。